docs/connect_servers/kubernetes/kubernetes.md

```diff
@@ -0,0 +1,76 @@
+# Kubernetes Clusters Management
+
+## Creating new Kubernetes cluster
+
+After launching servers with Talos Linux image generated by the Fluence Application, they will be staying in Maintenance mode. On this stage Talos is still waiting to be properly configured to host Kubernetes workload.
+
+To configure Talos Linux and launch K8s cluster, please navigate to the  Clusters tab and create your first cluster selecting the following  parameters:
+
+1. *Data center* -  Fluence software supports easily configurable cluster provisioning and management for servers located in a single physical data 
+2. *VLAN ID for private networking* - optional parameter that should be set if you have additional networking logic for internal traffic used  for Talos API and K8s API.
+3. *Kubernetes roles* - for each individual server, specify its [K8s role](https://kubernetes.io/docs/concepts/architecture/nodes/). The Fluence team recommends to follow default Kubernetes recommendation and support an odd number of Control plane nodes. To increase the stability of the Cluster, we recommend to have a minimum of  three (3) Control plane nodes and place them in different racks / cabinets within the Data center.
+:::danger **Attention**
+
+You should always keep a minimum of one (1) healthy Control plane node to support cluster integrity.  If all control plane nodes will be unavailable (due to the private networking or hardware issues) Kubernetes Cluster will be broken and won’t be able to restore his state.
+:::
+4. *Workload interface* - for each individual server, select the network interface that should be used for running workload from Fluence customers. This interface should be minimum 10Gb and 100Gb recommended.
+5. *Talos disk* - for each server specify a disk for the installation of  Talos Linux. Fluence recommends to use NVMe drives with at least 200GB free space and RAID1 as the system disk.
```

Comment:
100GB is enough according to Talos recommendations. Also NVME is not a requirement, SSD is more than enough for the system partition.